Library/Formula/llvm.rb

```diff
@@ -78,17 +101,27 @@ class Llvm < Formula
       url "http://llvm.org/git/lldb.git"
     end
 
-    resource "clang-tools-extra" do
-      url "http://llvm.org/git/clang-tools-extra.git"
+    resource "polly" do
+      url "http://llvm.org/git/polly.git"
+    end
+
+    resource "compiler-rt" do
```

Comment:
Just a detail, but now that it's called `compiler-rt` you probably want to move it up to keep the alphabetical order.